spec/std/http/http_spec.cr

```diff
@@ -37,15 +37,15 @@ describe HTTP do
     parsed_time.to_utc.to_s.should eq("2011-09-10 02:36:00 UTC")
   end
 
-  describe "generates RFC 1123" do
+  describe "generates HTTP date" do
     it "without time zone" do
       time = Time.utc(1994, 11, 6, 8, 49, 37, nanosecond: 0)
-      HTTP.rfc1123_date(time).should eq("Sun, 06 Nov 1994 08:49:37 GMT")
+      HTTP.format_time(time).should eq("Sun, 6 Nov 1994 08:49:37 GMT")
```

Comment:
The name was changed because the methods doesn't implement one exact standard format but generally format and parse time instances for use in a HTTP protocol context (see previous comments in https://github.com/crystal-lang/crystal/pull/4729#issuecomment-317831282).

I'll look into the preferred digit format.